CRT.c

```diff
@@ -1046,7 +1046,14 @@ void CRT_init(const Settings* settings, bool allowUnicode, bool retainScreenOnEx
       CRT_scrollHAmount = 5;
    }
 
-   if (termType && (String_startsWith(termType, "xterm") || String_eq(termType, "vt220"))) {
+   if (termType && (
```

Comment:
Can we add a section comment or factor this into a function on what this list of `termType` matches for?